File: API/Producteur.php
<?php

class Producteur {
    private $id;
    private $nom;
    private $prenom;
    private $tel;
    private $mail;
    private $mdp;

    function __construct( $_id, $_nom,$_prenom,$_tel,$_mail,$_mdp)
    {
        $this->id = $_id;
        $this->nom = $_nom;
        $this->prenom = $_prenom;
        $this->tel = $_tel;
        $this->mail = $_mail;
        $this->mdp = $_mdp;
    }

    public function getId() {
        return $this->id;
    }

    public function getNom() {
        return $this->nom;
    }

    public function getPrenom() {
        return $this->prenom;
    }

    public function getTel() {
        return $this->tel;
    }
    
    public function getMail() {
        return $this->mail;
    }
    
}

File: Android(Matetest)/app/src/main/java/com/example/matetest/Producteur.java
package com.example.matetest;

public class Producteur {

    private int id;
    private String nom;
    private String prenom;
    private String tel;
    private String mail;
    private String pass;

    public Producteur(int id,String nom, String prenom, String tel , String mail , String pass){
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.tel = tel;
        this.mail = mail;
        this.pass = pass;
    }

    public int getId(){
        return id;
    }
    public String getNom(){
        return nom;
    }
    public String getPrenom(){
        return prenom;
    }
    public String getTel(){
        return tel;
    }
    public String getMail(){
        return mail;
    }
    public String getPass(){
        return pass;
    }

}
